Example/Messaging/Sample/Sample/ContentView.swift

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 Google
```

Comment:
New copyrights should be `Google LLC`. See go/copyright

Reply:
Done.